docs/content/content/content-styles.mdx

```diff
@@ -193,13 +191,13 @@ Try not to use a Frequently Asked Question (FAQ) to ‘fix’ content that may b
 
 Reserve FAQs for ‘edge’ cases as much as possible. That is, user journeys and outcomes that happen rarely. Instead try to make content for all users clearer and more concise.
 
-Provide guidance and answers a user needs in the context of what they are doing. That is, at the point of their journey or within a transaction where they need a piece of information.
+Provide guidance and answers a user‘s need in the context of what they are doing. That is, at the point of their journey or within a transaction where they need a piece of information.
```

Comment:
This sentence doesn't make sense. Shouldn't have added the apostrophe s?

Reply:
- [ ] Update this sentence so that it's clearer.